frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -62,14 +63,14 @@ function updateSectionsWithPlugins(sections) {
 const SECTIONS = updateSectionsWithPlugins({
   setup: {
     name: t`Setup`,
-    order: 1,
+    order: 10,
```

Comment:
I bumped these all up x10 so that it's easier to add things in the future without renumbering everything.